src/issuer-helper.ts

```diff
@@ -11,30 +11,91 @@ import { Issuer } from 'openid-client';
 import { dbCredClient } from './database';
 import { getConfig } from './config';
 
+export enum AuthType {
+  OidcToken = 'OIDC_TOKEN',
```

Comment:
```suggestion
  OidcToken = 'oidc_token',
```
I think literal constant values are better in lowercase. (Uppercase, the scan like env variables. So, seeing something like `AUTH_TYPE = OIDC_TOKEN` in the docs, makes the user think "oh, assign the value of the OIDC_TOKEN env var, to AUTH_TYPE".)